Podverse/MediaPlayerViewController.swift

```diff
@@ -80,6 +80,11 @@ class MediaPlayerViewController: PVViewController {
         updateTime()
     }
     
+    // Adding togglePlayIcon in viewDidAppear and viewWillAppear since the icon was not always correct after returning from the background
```

Comment:
If you add the toggle on viewDidAppear, you don't need it in viewWillAppear as viewDidAppear it comes second in the lifecycle

Reply:
Well, if I remove it from viewWillAppear, then you can see the play / buffering / pause buttons all at once until viewDidAppear gets called.

I'll remove it from viewDidAppear since it seems like a bad fix for the "icon incorrect sometimes after returning from background mode". I should probably investigate this more...